apps/gallery/test/marionette/lib/fullscreen_view.js

```diff
@@ -255,7 +256,31 @@
         Fullscreen_View.Selector.shareButton);
     },
 
-      /**
+    /**
+     * @return {Marionette.Element} Element containing fullScreenView toolbar
+     */
+    get fullScreenToolBar() {
+      return this.client.helper.waitForElement(
+        Fullscreen_View.Selector.fullScreenToolBar);
+    },
+
+    /**
+    * @return {numeric} fullScreenToolBar width.
+    */
+    get ToolBarWidth() {
+      return this.fullScreenToolBar.size().width;
+    },
+
+    /**
+     * @return {Marionette.Element}  List of elements of toolbar options.
+     */
+    get ToolBarOptions() {
+      this.waitFor(Fullscreen_View.Selector.fullScreenToolBarOptions);
```

Comment:
Any specific reason to use ````this.waitFor()```` here? I know it is just ````this.client.helper.waitForElement()```` but then the latter is better known.